scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -46,6 +48,7 @@ struct layer layers[MAX_LAYERS];
 u32 fallback_cpu;
 static u32 preempt_cursor;
 
+static volatile uint32_t dsq_timer_check_flag = 0;
```

Comment:
Maybe use the same word consistently? ie. if the feature is antistall, use the word in the flag too?